jane/doc/extensions/_01-stack-allocation/reference.md

```diff
@@ -862,7 +862,7 @@ let a, b, c =
   x, y, z
 ```
 
-Again, there is no actual syntax for this in OCaml: that's a binding
+Again, there is no actual syntax for this in stock OCaml: that's a binding
```

Comment:
And here. Maybe the best way to address this is some kind of introduction explaining our terminology? But I worry no one would actually read that introduction.